### PR TITLE
add SapceAPI JSON #41

### PR DESCRIPTION
**Pull request by chris18890**
add SapceAPI JSON #41

[ISSUE-41](https://github.com/FarsetLabs/farsetlabs.github.io/issues/41)

## Description

Add SapceAPI JSON #41

Before | After
--- | ---
placeholder image | placeholder image

### CODE REVIEW

**Comment by awflwafl** [comment]
@chris18890 did you use some plugin for creating the pull request? The description is all messed up.

File: farsetlabs.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+    "api": "0.13",
+    "botname": "FarBot",
+    "spacefed": {
+        "spacenet": false,
+        "spacesaml": false,
+        "spacephone": false
+    },
+    "contact": {
+        "email": "admin@farsetlabs.org.uk",
```
Comment (awflwafl) [inline farsetlabs.json:10]:
```suggestion
        "email": "info@farsetlabs.org.uk",
```

File: farsetlabs.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        "flickr": "http://www.flickr.com/groups/farset_labs",
+        "instagram":"https://www.instagram.com/farsetlabs",
+        "irc": "irc://irc.freenode.net/#farsetlabs",
+        "phone": "+448458694619",
+        "discourse":"https://discourse.farsetlabs.org.uk"
+    },
+    "issue_report_channels": [
+        "email",
+        "twitter"
+    ],
```
Comment (awflwafl) [inline farsetlabs.json:24]:
```suggestion
    "issue_report_channels": [
        "email",
        "slack"
    ],
```

File: farsetlabs.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    "location": {
+        "address": "Weavers Court, Linfield Industrial Estate, Belfast, BT125GH, UK",
```
Comment (awflwafl) [inline farsetlabs.json:26]:
```suggestion
        "address": "1 Weavers Court, Linfield Industrial Estate, Belfast, BT125GH, UK",
```

File: farsetlabs.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        "lon": -5.94068,
+        "lat": 54.592813
+    },
+    "logo": "http://unit1.farsetlabs.org.uk/img/logo.png",
```
Comment (awflwafl) [inline farsetlabs.json:30]:
```suggestion
    "logo": "https://www.farsetlabs.org.uk/assets/img/logo.png",
```

File: farsetlabs.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    "state": {
+        "lastchange": 1333824073,
+        "open": true
+    },
+    "space": "Farset Labs",
+    "status": "Open",
+    "url": "http://www.farsetlabs.org.uk"
```
Comment (awflwafl) [inline farsetlabs.json:37]:
```suggestion
    "url": "https://www.farsetlabs.org.uk"
```

File: farsetlabs.json [hunk 6]
```diff
@@ -0,0 +1,38 @@
+{
+    "api": "0.13",
+    "botname": "FarBot",
```
Comment (awflwafl) [inline farsetlabs.json:3]: What's this thing? I don't think we've got a bot.

File: farsetlabs.json [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    "spacefed": {
+        "spacenet": false,
+        "spacesaml": false,
+        "spacephone": false
+    },
+    "contact": {
+        "email": "admin@farsetlabs.org.uk",
+        "github":"https://github.com/FarsetLabs",
+        "twitter": "@farsetlabs",
```
Comment (awflwafl) [inline farsetlabs.json:12]: Everything else in the `contact` sections is a URL should his also be a URL?

Comment (chris18890) [reply farsetlabs.json:12]: I htink ti will automatically format the twitter name as a URL/link, it just doesn't handle the others as well

File: farsetlabs.json (hunk 2 above)
Comment (chris18890) [reply farsetlabs.json:24]: when you test the file against https://spaceapi.io/validator/ it comes back with this error

{"valid":false,"message":"(root).issue_report_channels.1: issue_report_channels.1 must be one of the following: \"email\", \"issue_mail\", \"twitter\", \"ml\"\n"}

File: farsetlabs.json (hunk 6 above)
Comment (chris18890) [reply farsetlabs.json:3]: we _used_ to, (@andrewbolster ) removed for now :)

File: farsetlabs.json (hunk 1 above)
Comment (chris18890) [reply farsetlabs.json:10]: updated :)

File: farsetlabs.json (hunk 3 above)
Comment (chris18890) [reply farsetlabs.json:26]: updated :)

File: farsetlabs.json (hunk 4 above)
Comment (chris18890) [reply farsetlabs.json:30]: updated :)

File: farsetlabs.json (hunk 5 above)
Comment (chris18890) [reply farsetlabs.json:37]: updated :)

**Comment by chris18890** [comment]
> 
> 
> @chris18890 did you use some plugin for creating the pull request? The description is all messed up.

I just created the PR from within the Github Desktop app

File: farsetlabs.json (hunk 2 above)
Comment (awflwafl) [reply farsetlabs.json:24]: Ah, thanks. I didn't know about that.

File: farsetlabs.json (hunk 6 above)
Comment (awflwafl) [inline farsetlabs.json:3]:
```suggestion
```

File: farsetlabs.json (hunk 2 above)
Comment (awflwafl) [reply farsetlabs.json:24]:
```suggestion
    "issue_report_channels": [
        "email"
    ],
```

**Comment by awflwafl** [comment]
Happy to merge with the last 2 suggestions.

**Comment by awflwafl** [comment]
@andrewbolster 🙊 no comment.

**Pull request merged**